datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts

```diff
@@ -0,0 +1,74 @@
+import { EntityType, SchemaField } from '../../../../../../../types.generated';
+import EntityRegistry from '../../../../../EntityRegistry';
+
+// returns list of fieldPaths for fields that have Terms or Tags matching the filterText
+function getFilteredFieldPathsByMetadata(editableSchemaMetadata: any, entityRegistry, filterText) {
+    return (
+        editableSchemaMetadata?.editableSchemaFieldInfo
+            .filter((fieldInfo) => {
+                return (
+                    fieldInfo.globalTags?.tags.find((tagAssociation) =>
+                        entityRegistry
+                            .getDisplayName(EntityType.Tag, tagAssociation.tag)
+                            .toLocaleLowerCase()
+                            .includes(filterText),
+                    ) ||
+                    fieldInfo.glossaryTerms?.terms.find((termAssociation) =>
+                        entityRegistry
+                            .getDisplayName(EntityType.GlossaryTerm, termAssociation.term)
+                            .toLocaleLowerCase()
+                            .includes(filterText),
+                    )
+                );
+            })
+            .map((fieldInfo) => fieldInfo.fieldPath) || []
+    );
+}
+
+function shouldInclude(
+    fieldName: string,
+    fullFieldPath: string,
+    filterText: string,
+    filteredFieldPathsByMetadata: any,
+) {
+    return fieldName.toLocaleLowerCase().includes(filterText) || filteredFieldPathsByMetadata.includes(fullFieldPath);
```

Comment:
are we also toLowerCasing filterText?

Reply:
yup! on line 45 right when we call `filterSchemaRows` i lowercase it and ass that around to these functions and other places so it's always lowercase comparisons